services/tenant-ui/config/default.json

```diff
@@ -8,6 +8,11 @@
     "showInnkeeperAdminLogin": true,
     "showOIDCReservationLogin": false,
     "showWritableComponents": true,
+    "tokenExpirySeconds": "86400",
```

Comment:
Is this meant to sync up with the ACA-Py token lifespan? If so we can have those pull from the same environmental config I _think_. But that will be totally in the helm chart realm, this default field would remain.

Reply:
Yes it was supposed to link up. I'm not sure if this feature is needed anymore. I added it before I added forcing the user to login again on a new tab/window. If the token expired they would get "randomly" logged out and wanted to tell them why. But now the user would need to use the app for a day straight without the inactivity log out.

I think I may remove this feature.

Reply:
This is removed now